webhook/mutate/webhook.go

```diff
@@ -0,0 +1,223 @@
+package mutate
+
+import (
+	"encoding/json"
+	"errors"
+	"fmt"
+	"io/ioutil"
+	"log"
+	"net/http"
+
+	v1 "k8s.io/api/admission/v1"
+	corev1 "k8s.io/api/core/v1"
+	metav1 "k8s.io/apimachinery/pkg/apis/meta/v1"
+)
+
+// CoordAddr contains the address of the marblerun coordinator
+var CoordAddr string
```

Comment:
Please create a mutator object (struct) for this, which is passed to the http.mux and used from there.
See the clientAPI server for reference.